src/apps/common/submodules/navigationWizard/navigationWizard.scss

```diff
@@ -172,3 +172,41 @@
 	background: #2ED022;
 	color: #071F05;
 }
+
+@media print {
+	.core-topbar-wrapper {
+		display: none;
+	}
```

Comment:
I believe it would make more sense for this selector to be declared in `core/style/app.scss`, with a dedicated media query.